### PR TITLE
Add Gothic keyboard

### PR DESCRIPTION
**Pull request by amire80**

### CODE REVIEW

**Comment by amire80** [comment]
This is a continuation of #448 by @bokareis.

In this pull request I did the things requested by @santhoshtr - squashing and adding tests. I also did some more cleanup, and syntax correction.

However, this cannot yet be merged yet, because it seems to have logic problems with the context, particularly for letters 𐍈 and 𐌸. Which letter sequences are supposed to produce them? It's not clear from the code.

**Comment by bokareis** [comment]
Hello,

Thanks for doing the necessary things which I couldn't do!

𐍈 represents the hwair, it uses the letter combination h + w. If it is possible to have 2 letter combinations, you could add h + v as an extra, both ways can be used, but most often it is h + w.

𐌸 represents the character thorn: þ. It uses the combination t+h.

**Comment by amire80** [comment]
OK, so can it ever happen that "𐌷𐍅" or "𐍄𐌷" is written? Or is it supposed to be always replaced with 𐍈 and 𐌸?

Can I suggest something?

- Currently "v" and "w" both produce "𐍅". Maybe "v" can produce "𐍅" and "w" can produce "𐍈"?
- There are no capital letters. Perhaps "T" can produce "𐌸"?

This way we can avoid double letters, so it's easier to document, implement, and test.

**Comment by amire80** [comment]
Beep :)

I'd like to complete the work on this, so if I'm not getting an answer until June 20, I'll go forward with my suggestions above, and it can be fixed later ;)

**Comment by bokareis** [comment]
Hello Amire,

I just realized there is a problem. I suggested to convert "t + h" to "þ" and "h + w" to "hw (symbol)", but someone might write a word combination like "athaitands", in which t and h are seperate and shouldn't be converted to þ. Your idea for a capital T is reasonable, and a work-around for the cases in which "th" should be written instead of "þ". Maybe "D" could be added too.


- Maybe "v" can produce "𐍅" and "w" can produce "𐍈"? -> It is preferable if you do it the other way around, so "w" produces "𐍅", while "hv" produces "𐍈". I have looked it up, and where as you have words in which the combination "hw" is used, without using this special symbol, it isn't the case for "hv".

**Comment by bokareis** [comment]
You can also use http://www.wulfila.be/gothic/browse/search/ to see which letter combinations return results and can cause problems.

**Comment by amire80** [comment]
OK, I did something really simple:
* Capital T produces 𐌸
* v produces 𐍈

This is it: 26 lowercase letters produce one Gothic letter each, and the uppercase T produces 𐌸. No double letters. Is this good to go?

File: rules/got/got-standard.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+( function ( $ ) {
+	'use strict';
+
+	var gotStandard = {
+		id: 'got-standard',
+		name: '𐌲𐌿𐍄𐌹𐍃𐌺𐌰 𐍂𐌰𐌶𐌳𐌰',
+		description: 'Gothic keyboard layout',
+		date: '2016-27-06',
```
Comment (kartikm) [inline rules/got/got-standard.js]: Are we following YYYY-DD-MM format? :)

File: rules/got/got-standard.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		URL: 'http://github.com/wikimedia/jquery.ime',
+		author:  'Bokareis',
```
Comment (kartikm) [inline rules/got/got-standard.js]: Extra space after :

**Comment by amire80** [comment]
Issues addressed.

**Pull request merged**